frontend/src/produto/Produto.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import './Produto.css';
+import  { useState } from "react";
+
+
+var prod = function Produto(props){
+
+    const [showDetalhes, setShowDetalhes] = useState(0)
+
+    const [Edit, setEdit] = useState(0)
+
+
+
+    
+    function detalhes(){
+
+       if(showDetalhes === 1){
+          var data = new Date(props.data_cadastro)
+
+            if(data.getDate()>9){
+               var data_cadastro = ""+ data.getDate()+"/"
+             }else{
+               var data_cadastro = "0"+ data.getDate()+"/"
+            }
+
+    if((data.getMonth() + 1)>9){
+
+         data_cadastro += ""+ (data.getMonth()+1)+"/"
+         
+            }else{
+                 data_cadastro += "0"+ (data.getMonth()+1)+"/"
+            }
```

Comment:
Cuidado com a indentação do código.

Isso pode levar a vários erros de lógica pois não se encontra o final do bracket na mesma coluna do início do if.